python/lsst/ip/diffim/getTemplate.py

```diff
@@ -297,12 +297,18 @@ def run(self, *, coaddExposureHandles, bbox, wcs, dataIds, physical_filter):
                 self.log.info("No overlap from coadd patches in tract %s; not including in output.", tract)
                 continue
 
-            catalogs.append(catalog)
+            # Trim the exposure catalog to just the patches that were used.
+            temp = afwTable.ExposureCatalog(self.schema)
```

Comment:
Please use something like `tempCatalog` instead of just `temp` for the variable name.